stats/cloud/data.go

```diff
@@ -25,47 +25,28 @@ import (
 	"fmt"
 	"math"
 	"sort"
-	"strconv"
 	"time"
 
 	"github.com/loadimpact/k6/lib/metrics"
 	"github.com/loadimpact/k6/lib/netext/httpext"
 	"github.com/loadimpact/k6/stats"
 )
 
-const DataTypeSingle = "Point"
-const DataTypeMap = "Points"
-const DataTypeAggregatedHTTPReqs = "AggregatedPoints"
-
-// Timestamp is used for sending times encoded as microsecond UNIX timestamps to the cloud servers
-type Timestamp time.Time
+//noling:golint
+const (
+	DataTypeSingle             = "Point"
+	DataTypeMap                = "Points"
+	DataTypeAggregatedHTTPReqs = "AggregatedPoints"
+)
 
-// Equal will return true if the difference between the timestamps is less than 1 microsecond
-func (ct Timestamp) Equal(other Timestamp) bool {
-	diff := time.Time(ct).Sub(time.Time(other))
-	return diff > -time.Microsecond && diff < time.Microsecond
-}
+//go:generate easyjson -pkg -no_std_marshalers .
```

Comment:
hmm I can't run `go generate ./...` :confused: I get
```
easyjson-bootstrap443749265.go:12:3: cannot find package "." in:
	/rw/home/go/src/github.com/loadimpact/k6/vendor/github.com/mailru/easyjson/gen
Bootstrap failed: exit status 1
data.go:42: running "easyjson": exit status 1
```

I'm no sure if I've messed up my easyjson installation or something we messed up with moving to Go modules?

Reply:
I think this is the problem https://github.com/golang/mock/issues/415#issuecomment-602547154 ... 
I am somewhat on the fence about whether to use `-gen_build_flags -mod=mod` which works for us without additional imports or to add the `gen` library ?!?

Reply:
Not sure I completely understand the problem, but given that you need the `easyjson` _binary_ to build this anyway, I don't really care if we artificially add some extra things in vendor or not... If `-gen_build_flags -mod=mod` would always work, but needs to download some stuff from the internet, it's not a big deal, as long as `go generate ./...` works consistently. If we ever want to start running it in CI, we first have to fix some other issues with go-rice, so no preference from my side.

Reply:
I will go with the `-mod=mod` ... while it has chances to break (as it doesn't depend on what we vendor) this still requires someone to commit it .. .and it will break in "this can't compile" way, which I think is fine

Reply:
resolved in b04d813